.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,75 @@
+name: Bug Report
+description: Create a report to help us improve Xournal++.
+labels: bug
+body:
+  - type: dropdown
+    attributes:
+      label: Operating System
+      description: Select the operating system that you use.
+      options:
+        - Windows
+        - Mac OS
+        - Linux
+    validations:
+      required: true
+  - type: input
+    attributes:
+      label: (Linux only) Distribution
+      description: What distribution of Linux do you use?
+      placeholder: e.g. Ubuntu, Arch Linux, etc.
+  - type: input
+    attributes:
+      label: (Linux only) Desktop Environment
+      description: What desktop environment do you use?
+      placeholder: e.g. GNOME, KDE Plasma, etc.
+  - type: dropdown
+    attributes:
+      label: (Linux Only) Display Server
+      description: What display server do you use?
+      options:
+        - X.Org
+        - Wayland
+  - type: input
+    attributes:
+      label: Installation Method
+      description: How was Xournal++ installed?
+      placeholder: e.g. Official Release, Flatpak, PPA, etc.
```

Comment:
```suggestion
      placeholder: e.g. Official Release, Flatpak, Snap package, PPA, etc.
```

Reply:
Out of curiosity, why would you like "Snap" to be stated explicitly, and not be within "etc."? When writing that list, I figured that I would try to have 1 of every type of package format -- 1 representing an official repo's package, Flatpak representing the distro-agnostic package managers (like snap), and PPA representing the user-manageds repos similar to the AUR.

Reply:
Well then, leave `Snap package` away. I just thought that there are lots of snap package users (although probably not as many as flatpak users) for Xournal++.

Reply:
I'll add "Snap".

```suggestion
      placeholder: e.g. Official Release, Flatpak, Snap, PPA, etc.
```